source/glimmer-components/index.md

```diff
@@ -0,0 +1,37 @@
+## HTML elements as an abstraction for text documents
+
```

Comment:
Anything more belongs in dedicated guides: transition from Angular, transition from React, transition from Ember curlies.